tracing/otlp.go

```diff
@@ -0,0 +1,35 @@
+package tracing
+
+import (
+	"go.opentelemetry.io/otel/exporters/otlp"
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+	"google.golang.org/grpc/credentials"
+)
+
+// OTLP service to export traces to
+type OTLP struct {
+	Address string            `long:"otlp-address" description:"otlp address to send traces to"`
+	Headers map[string]string `long:"otlp-header" description:"headers to attach to each tracing message"`
+}
+
+// IsConfigured identifies if an Address has been set
+func (s OTLP) IsConfigured() bool {
+	return s.Address != ""
+}
+
+// Exporter returns a SpanExporter to sync spans to OTLP
+func (s OTLP) Exporter() (export.SpanSyncer, error) {
+	secureOption := otlp.WithTLSCredentials(credentials.NewClientTLSFromCert(nil, ""))
```

Comment:
Can it be configurable with an insecure option? It would make sidecars and local testing much less painful

Reply:
I just realized that this would mean we need to be able to pass in TLS certs as well, I'm sure there's some scenario where the tracing/metrics service is on a private network with self signed certs.

Reply:
for now it would be ok to just have an 'enable tls' flag, we can do custom certs in a later iteration